app/src/main/java/org/tokend/template/features/dashboard/balances/view/AssetDistributionChart.kt

```diff
@@ -118,6 +120,8 @@ class AssetDistributionChart
             override fun onValueSelected(e: Entry, h: Highlight) {
                 chart.centerText = percentFormatter.format(e.y / 100)
                 prevHighlightValue = h.x
+                selectedItemIndex = chart.data.dataSet.getEntryIndex((e as PieEntry))
```

Comment:
For pie chart entry index is it's X value so cast x to int and pass it to the legend display method without extra global variable